Model/Config/Backend/Enable.php

```diff
@@ -50,24 +50,14 @@ public function afterSave()
     {
         $isV2Enabled = (int) $this->getValue();
         if ($isV2Enabled === 0) {
-            $this->writer->save(\Extend\Integration\Service\Extend::ENABLE_PRODUCT_PROTECTION, 0);
-            $this->writer->save(\Extend\Integration\Service\Extend::ENABLE_SHIPPING_PROTECTION, 0);
-            $this->writer->save(
-                \Extend\Integration\Service\Extend::ENABLE_PRODUCT_PROTECTION_CART_OFFER,
-                0
-            );
-            $this->writer->save(
-                \Extend\Integration\Service\Extend::ENABLE_PRODUCT_PROTECTION_PRODUCT_DISPLAY_PAGE_OFFER,
-                0
-            );
-            $this->writer->save(
-                \Extend\Integration\Service\Extend::ENABLE_PRODUCT_PROTECTION_PRODUCT_CATALOG_PAGE_MODAL_OFFER,
-                0
-            );
-            $this->writer->save(
-                \Extend\Integration\Service\Extend::ENABLE_PRODUCT_PROTECTION_POST_PURCHASE_LEAD_MODAL_OFFER,
-                0
-            );
```

Comment:
removing these because we no longer need them to be dependent on the overall toggle - no offers will display if the global Enable toggle is `off`.

Reply:
How does the code handle individual elements being turned off?

Reply:
i think we covered this in another comment, but when the admin toggles off a given offer type, the corresponding phtml template's rendering logic will handle `rendering` or `not rendering` the offer. does that answer the question?